app/src/org/commcare/preferences/HiddenPreferences.java

```diff
@@ -175,6 +178,12 @@ public static boolean isSavedFormsEnabled() {
                 properties.getString(ENABLE_SAVED_FORMS, PrefValues.YES).equals(PrefValues.YES);
     }
 
+    public static boolean isLoggingEnabled() {
+        SharedPreferences properties = CommCareApplication.instance().getCurrentApp().getAppPreferences();
+        return !properties.contains(LOGS_ENABLED) ||
+                properties.getString(ENABLE_SAVED_FORMS, LOGS_ENABLED_YES).equals(LOGS_ENABLED_YES);
```

Comment:
I assume `ENABLE_SAVED_FORMS` should be `LOGS_ENABLED`?

Reply:
Also this needs to be checked before this line too:
https://github.com/dimagi/commcare-android/blob/20e0f0b5aec618eb88ff3413b3c6338ddca60b85/app/src/org/commcare/CommCareApplication.java#L175-L175

Reply:
It does not because the application has not been initialized at this point so there's nothing to check